generators/plugin/templates/composer.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "<%= plugin.name.fileName %>",
+  "type": "application",
+  "description": "<%= plugin.description %>",
+  "version": "1.0.0",
+  "homepage": "<%= plugin.website %>",
+  "authors": [
+    {
+      "name": "<%= plugin.author.name %>",
+      "email": "<%= plugin.author.email %>",
+      "homepage": "<%= plugin.author.website %>",
+      "role": "Developer"
+    }
+  ],
+  "config": {
+    "secure-http": false
```

Comment:
I'm not entirely convinced that setting this as a default is considered a good idea, since they fixed that old Composer bug where package downloads failed because of an SSL issue at wpackagist. Since wpackagist is used for the majority of WordPress plugins and themes, I would say adding that package repository explicitly would be a good idea.

```json
"repositories": [
    {
      "type": "composer",
      "url": "https://wpackagist.org"
    }
]
```